android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -37,7 +37,20 @@ function init() {
   keyman.beepKeyboard = beepKeyboard;
 
   // Readies the keyboard stub for instant loading during the init process.
-  KeymanWeb.registerStub(JSON.parse(jsInterface.initialKeyboard()));
+  try {
+    let starterStubStr = jsInterface.initialKeyboard();
```

Comment:
I vaguely remember Android 5.0 forcing us to use `var` instead of `let`

Reply:
Fixed in Chrome 49. https://caniuse.com/let

Reply:
Pretty sure that was an ES5 vs ES6 limitation.  Chrome 49+ is fine with let, and we currently state a minimum of Chrome 53.

https://github.com/keymanapp/keyman/blob/486807fd7d3fe09390154837648093331f85f921/android/docs/help/about/system-requirements.md?plain=1#L11-L15